pyproject.toml

```
@@ -9,7 +9,7 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
 streamdeck = "^0.9.3"
-pillow = "^9.4.0"
+pillow = "^10.0.0"
```

I don't think you necessarily need to require 10+, textlength and textbbox were added in 8.0.0: 

* https://pillow.readthedocs.io/en/stable/releasenotes/8.0.0.html#api-additions
* https://github.com/python-pillow/Pillow/pull/4959

```suggestion
pillow = "^9.4.0"
```